ang/workflow/services/duplicate-workflow-case-management.service.js

```diff
@@ -0,0 +1,24 @@
+(function (_, angular) {
+  var module = angular.module('workflow');
+
+  module.service('DuplicateWorkflowCasemanagement', DuplicateWorkflow);
+
+  /**
+   * Duplicate Case management workflows service
+   *
+   * @param {Function} civicaseCrmApi civicrm api service
+   */
+  function DuplicateWorkflow (civicaseCrmApi) {
+    this.create = create;
+
+    /**
+     * @param {object} workflow workflow object
+     * @returns {Array} api call parameters
+     */
+    function create (workflow) {
+      return civicaseCrmApi([
+        ['CaseType', 'create', _.extend(workflow, { id: null })]
```

Comment:
try not to send the original object as the first parameter when using `_.extend` since it will replace the original values:

![Screen Shot 2020-10-28 at 7 55 57 AM](https://user-images.githubusercontent.com/1642119/97432866-07af9880-18f3-11eb-86d5-a9eec32c9255.png)

Use `_.extend({}, workflow, { id: null })` instead.